extras/docker/initDockerLicode.sh

```diff
@@ -1,7 +1,167 @@
 #!/usr/bin/env bash
+SCRIPT=`pwd`/$0
+ROOT=/opt/licode
+SCRIPTS="$ROOT"/scripts
+BUILD_DIR="$ROOT"/build
+DB_DIR="$BUILD_DIR"/db
+EXTRAS="$ROOT"/extras
+NVM_CHECK="$ROOT"/scripts/checkNvm.sh
+
+parse_arguments(){
+  if [ -z "$1" ]; then
+    echo "No parameters -- starting everything"
+    MONGODB=true
+    RABBITMQ=true
+    NUVE=true
+    ERIZOCONTROLLER=true
+    ERIZOAGENT=true
+    BASICEXAMPLE=true
+
+  else
+    while [ "$1" != "" ]; do
+      case $1 in
+        "--mongodb")
+        MONGODB=true
+        ;;
+        "--rabbitmq")
+        RABBITMQ=true
+        ;;
+        "--nuve")
+        NUVE=true
+        ;;
+        "--erizoController")
+        ERIZOCONTROLLER=true
+        ;;
+        "--erizoAgent")
+        ERIZOAGENT=true
+        ;;
+        "--basicExample")
+        BASICEXAMPLE=true
+        ;;
+      esac
+      shift
+    done
+  fi
+}
+
+run_nvm() {
+  echo "Running NVM"
+  . $ROOT/build/libdeps/nvm/nvm.sh
+
+}
+check_result() {
+  if [ "$1" -eq 1 ]
+  then
+    exit 1
+  fi
+}
+run_rabbitmq() {
+  echo "Starting Rabbitmq"
+  rabbitmq-server -detached
+  sleep 3
+}
+
+run_mongo() {
```

Comment:
I agree that we're duplicating code here, but I would do it as a part of a bigger refactoring of the scripts. Right now the populating the database is tied to copying the configuration and that can cause all sorts of problems if we make it an option in the default scripts.